activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+*   Run transactional callbacks on the freshest instance to save a given 
+    record within a transaction.
+
+    When multiple Active Record instances change the same record within a 
+    transaction, Rails runs `after_commit` or `after_rollback` callbacks for 
+    only one of them. This adds an option to specify how Rails chooses which 
```

Comment:
This part needs to be written in the past tense since CHANGELOGs entries are usually read after a release.

Something like:

```
`config.active_record.run_commit_callbacks_on_first_saved_instances_in_transaction` was added to specify how Rails chooses which instance receives the callbacks. The framework default was changed to defaults to the new
logic.
```